sensor/debugger/k8s/fake.go

```diff
@@ -436,6 +621,68 @@ func (f *FakeEventsManager) waitOnMode(events []string) error {
 	return nil
 }
 
+// execFuncToAllResourcesOfKind executes a given function to all resources of a given kind
```

Comment:
I understand that maybe the goal was to provide a callback to each resource object . However, only the name seems passed. Which is enough for removing namespaces. I suggest rewording this.
Additionally, you could already provide an `interface{}` object parameter to the `execFunc` callback. 

IMO this would read better:
```suggestion
// forEachResourceName iterates on every resource of given kind and call `callbackFn` for each resource object name
```
Or `forEachResource` if you decide on passing the entire object to the callback.

Reply:
Keep in mind that if you decide to pass the entire object you could remove the`GetName()` reflection above. It would be the responsibility of the caller to cast the `interface{}` into whatever resource is being listed.

Reply:
Done. I've decided to pass the reflected value of the resource instead of the name. Let me know what you think.